### DISCUSSION TITLE
自定义路由无效

### DISCUSSION THREAD

**Discussion by AmberisMyShiba** [SagerNet/SagerNet#113] [Q&A] [answered]
**Describe the bug**
添加自定义绕行路由无效
**To Reproduce**
1. Go to '...'路由
2. Click on '....'新建路由
3. Scroll down to '....'添加ip，这里以45.32.164.128为例。观察ip111.cn从国外测试的结果
4. See error'从国外测试的结果显示，是不是直连地址而是代理地址'

**Logcat**
Export it yourself :)

**Expected behavior**
添加一个绕行的IP后，ip111.cn从外国测试应该显示的是直连地址
**Screenshots**
![Screenshot_20210525-144321__01](https://user-images.githubusercontent.com/20527882/119458534-706ad080-bd6f-11eb-8475-ff2773d07915.jpg)
![Screenshot_20210525-144321__02__01](https://user-images.githubusercontent.com/20527882/119458543-72cd2a80-bd6f-11eb-9262-48e1b060bd37.jpg)

**Smartphone (please complete the following information):**
 - Device: [e.g. oneplus8T]
 - OS: [e.g. Android11]
 - Version [e.g. 5.15.0]

**Additional context**

**Comment by nekohasekai** [comment, marked as answer]
按住往上拉, 以及, 你的写法本身错误, 多去了解文档.
